tests/fix_tmp_dir_session_test_util.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+from .fixtures import *  # noqa
+
+dir = None
+
+
+def test_1(tmp_dir_session):
```

Comment:
please name tests including the name of the thing tested. (If very long can be abbreviated.) In current case: `test_tmp_dir_session_*` ... suffix should give some indication of scenario tested. Numbers aren't completely ruled out but frowned on.

Also you can't count on tests running in sequence. This first test -- since it has no asserts -- should actually be a fixture. Could be `fix_tmp_dir_session_with_file`. Or is short enough to combine with test.

If fixture, should write `uhh` and yield in try; then in finally clause get rid of `uhh` so session repo is clean. Otherwise, other users (perhaps in other test files) of `tmp_dir_session` will see `uhh`.

So -- in fact -- best to be fixture as it does need this cleanup step. Remember that `tmp_dir_session` is for the whole session including all other tests.